exampleRios.py

```diff
@@ -0,0 +1,131 @@
+from backtesting import evaluateHist
+from strategy import Strategy
+from order import Order
+import numpy as np
+
+class RSI(Strategy):
```

Comment:
@marcelogdeandrade has already done RSI, you should have done your research before working.

Reply:
I don't care, this in not an impediment.